softdevice_controller/CHANGELOG.rst

```diff
@@ -46,14 +46,17 @@ Bugfixes
 Known issues and limitations
```

Comment:
Didn't we decide to rename this to just "Limitations"? 
(Or maybe even Limitation, since it's only one.)

Reply:
We may rename it to "Limitations". Known issues are are placed in the NCS known issue page